src/Interfaces.ts

```diff
@@ -29,17 +45,17 @@ export enum ClaimType {
   Work = 'Work',
 }
 
-export interface Work extends Claim<WorkAttributes> {}
-
 export interface WorkAttributes extends ClaimAttributes {
   readonly name: string
   readonly datePublished: string
   readonly dateCreated: string
```

Comment:
These date types look correct. Why did our test code try to use a JS date object?